buildpack/infrastructure/database.py

```diff
@@ -303,10 +335,11 @@ def init(self):
         if database_type == "PostgreSQL" and config["DatabaseHost"].split(":")[
             0
         ].endswith(".rds.amazonaws.com"):
+            database_region = config["DatabaseHost"].split('.')[2]
             jdbc_params.update(
                 {
                     "sslrootcert": os.path.expandvars(
-                        "$HOME/.postgresql/amazon-rds-ca.pem"
+                        "$HOME/.postgresql/"+region_pem_map[database_region]
```

Comment:
Why not something like `f"$HOME/.postgresql/{database_region}-bundle.pem"`?

Reply:
We did discuss this because we need to import/write the pem files to the /etc manually to keep the build pack offline. so we wanted to make it more explicit what we are doing.